dom/events/non-cancelable-when-passive/synthetic-events-cancelable.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<title>Synthetic events are always cancelable by default</title>
+<link rel="help" href="https://dom.spec.whatwg.org/#dictdef-eventinit">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+const eventsMap = {
+  wheel: 'WheelEvent',
+  mousewheel: 'WheelEvent',
+  touchstart: 'TouchEvent',
+  touchmove: 'TouchEvent',
+  touchend: 'TouchEvent',
+  touchcancel: 'TouchEvent',
+}
+function isCancelable(eventName, interfaceName) {
+  test(() => {
+    assert_implements(interfaceName in self, `${interfaceName} should be supported`);
+    let defaultPrevented = null;
+    addEventListener(eventName, event => {
+      event.preventDefault();
+      defaultPrevented = event.defaultPrevented;
+    });
+    const event = new self[interfaceName](eventName);
+    assert_false(event.cancelable, 'cancelable');
+    const dispatchEventReturnValue = dispatchEvent(event);
+    assert_false(defaultPrevented, 'defaultPrevented');
+    assert_true(dispatchEventReturnValue, 'dispatchEvent() return value');
+  }, `Synthetic ${eventName} event with interface ${interfaceName} is cancelable`);
```

Comment:
Isn't this showing the event is *not* cancelable?

Reply:
Oops
```suggestion
  }, `Synthetic ${eventName} event with interface ${interfaceName} is not cancelable`);
```